GameDefinition/game-src/Client/UI/Content/Input.hs

```diff
@@ -75,10 +75,10 @@ standardKeysAndMouse = InputContentRaw $ map evalKeyDef $
   -- Item menu, first part of item use commands
   , ("comma", grabItems "")  -- only show extra key, not extra entry
   , ("r", dropItems "remove item(s)")
-  , ("f", addCmdCategory CmdItemMenu $ projectA flingTs)
+  , ("f", addCmdCategory CmdItemMenu $ projectA [])
   , ("C-f", addCmdCategory CmdItemMenu
             $ replaceDesc "auto-fling and keep choice"
-            $ projectI flingTs)
+            $ projectI [])
```

Comment:
I'd hope you can remove the argument altogether from projectI --- the whole list argument

Reply:
just as you removed it from `projectICmd`